File: 412.fizz-buzz.py
#
# @lc app=leetcode id=412 lang=python
#
# [412] Fizz Buzz
#

# @lc code=start
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        returnArray = []
        for i in range(1,n+1):
            if i%15 == 0:
                returnArray.append("FizzBuzz")
            elif i%5 == 0:
                returnArray.append("Buzz")
            elif i%3 == 0:
                returnArray.append("Fizz")
            else:
                returnArray.append(str(i))
        return returnArray        
        
# @lc code=end

File: 412.fizz-buzz.java
/*
 * @lc app=leetcode id=412 lang=java
 *
 * [412] Fizz Buzz
 */

// @lc code=start
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> returnArray = new ArrayList<>();
        for (int i = 1; i <= n; i++) {
        if (i%15==0) {
            returnArray.add("FizzBuzz");
        }
        else if (i%5 == 0) {
            returnArray.add("Buzz");
        }
        else if (i%3 == 0) {
            returnArray.add("Fizz");
        }
        else {
            returnArray.add(Integer.toString(i));
        }
        }
        return returnArray;            
    }
}
// @lc code=end

